src/utils.js

```diff
@@ -1,9 +1,17 @@
-import { castPath } from './key-path'
+/**
+ * @flow
+ */
```

Comment:
I know DocumentationJS could generate JSDoc meta data from Flow types. My initial thoughts were to do something like:

```
/**
 * @fileoverview ....
 * @flow
 */
```